graspologic/align/seeded_procrustes.py

```diff
@@ -0,0 +1,186 @@
+    The global objective function is :math:`|| X Q - P Y ||_F`.
+
+    Note that both :math:`X` and :math:`PY` are matrices in :math:`M_{n, d}`.
+    Thus, if one knew :math:`P`, it would be simple to obtain an estimate for
+    :math:`Q`, using the regular orthogonal procrustes. On the other hand, if
+    :math:`Q` was known, then :math:`XQ` and :math:`Y` could be thought of
+    distributions over a finite number of masses, each with weight :math:`1/n`
+    or :math:`1/m`, respectively. These distributions could be "matched" via
+    solving an optimal transport problem.
+
+    However, both :math:`Q` and :math:`P` are simultaneously unknown here, so
+    the algorithm performs a sequence of alternating steps, obtaining
+    iteratively improving estimates of :math:`Q` and :math:`P`, similarly to an
+    expectation-maximization (EM) procedure. It is not known whether this
+    procedure is formally an EM, but the analogy can be drawn as follows: after
+    obtaining an initial guess of of :math:`\hat{Q}_{0}`, obtaining an
+    assignment matrix :math:`\hat{P}_{i+1} | \hat{Q}_{i}` ("E-step") is done by
+    solving an optimal transport problem via Sinkhorn algorithm, whereas
+    obtaining an orthogonal alignment matrix :math:`Q_{i+1} | P_{i}`
+    is done via regular orthogonal procurstes. These steps are further simplified
+    by the use of a seeded matrix that contains rows of seeds that align a set of
+    indeses in ``X`` and ``Y``.
+    """
+
+    def fit(
+        self,
+        X,
+        Y,
+        seeds,
+        verbose=False,
+    ) -> "SeededProcrustes":
+
+        """
+        Overrides the fit method from the parent class ``BaseAlign`` to find
+        a value for ``Q`` given two datasets, ``X`` and ``Y``, and a matrix of seeds.
+
+        Parameters
+        ----------
+        X : np.ndarray, shape (n, d)
+            Dataset to be mapped to ``Y``, must have same number of dimensions
+            (axis 1) as ``Y``.
+
+        Y : np.ndarray, shape (m, d)
+            Target dataset, must have same number of dimensions (axis 1) as ``X``.
+
+        seeds : np.ndarray, shape (?,2)
+            Matrix of pairs, demonstrates the relation between rows of ``X`` and ``Y``.
+            The first column pertains to the ``X`` component of the pairs, and the second column
+            pertains to the ``Y`` component. The pairs are stored in the rows of ``seeds``,
+            each row containing an ``X`` value and a corresponding ``Y`` value.
+
+        verbose : bool
+            Degree to which the code records and produces feedback when run.
+
+        Returns
+        -------
+        self : returns an instance of self
+        """
+
+        n = len(X[0])
+        init_Q = _find_Q_from_pairs(X, Y, seeds)
+        procruster = SeedlessProcrustes(
+            init="custom",
+            initial_Q=init_Q,
+            optimal_transport_eps=1.0,
+            optimal_transport_num_reps=100,
+            iterative_num_reps=10,
```

Comment:
can you make these parameters of the class in the constructor? can copy names/docs from seedless